_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -73,9 +66,7 @@ template<class T> inline void Zero(T & obj)                   { memset(&obj, 0,
 template<class T> inline void Zero(std::vector<T> & vec)      { memset(&vec[0], 0, sizeof(T) * vec.size()); }
 template<class T> inline void Zero(T * first, size_t cnt)     { memset(first, 0, sizeof(T) * cnt); }
 template<class T> inline T Abs  (T x)               { return (x > 0 ? x : -x); }
-template<class T> inline T Min  (T x, T y)          { return MFX_MIN(x, y); }
-template<class T> inline T Max  (T x, T y)          { return MFX_MAX(x, y); }
-template<class T> inline T Clip3(T min, T max, T x) { return Min(Max(min, x), max); }
+template<class T> inline T Clip3(T minval, T maxval, T x) { return std::min(std::max(minval, x), maxval); }
```

Comment:
I think I saw this clip thing in 2 or 3 commits under other names! Arrgggh! I WANT normal core in mediasdk with utilities.

Reply:
yes... moreover, it came to std in c++17 as ```std::clamp```

Reply:
Here is #926. I suggest to introduce mfx::clamp before merging in this patch for min/max.